src/api/libcellml/units.h

```diff
@@ -417,47 +417,55 @@ class LIBCELLML_EXPORT Units: public NamedEntity, public ImportedEntity
      */
     size_t unitCount() const;
 
+    /**
+     * @brief Check whether there are any imported child @c Units.
+     *
+     * @return @c true when these @c Units rely on @c Units which are imported,
+     * or @c false otherwise.
+     */
+    bool requiresImports() const;
```

Comment:
I am not sure I would make this method part of the API, not least because it's only being used internally.

Reply:
I can imagine it being useful when you parse a model and want to check whether it depends on any imported instances beyond what you've already got? I'll add a test to show it.

Reply:
```
TEST(Units, parsedUnitsRequiringImports)
{
    auto parser = libcellml::Parser::create();
    auto model = parser->parseModel(fileContents("requires_imports.cellml"));

    // When parsing a model it's hard to know whether the units it uses are imported
    // or not if the import is in a child units item.  Using the Units::requiresImports()
    // function should tell us.

    EXPECT_EQ(size_t(3), model->unitsCount());
    EXPECT_FALSE(model->units("grandpa")->isImport());
    EXPECT_TRUE(model->units("grandpa")->requiresImports());
}
```